contents/product-engineers/pmf-surveys.md

```diff
@@ -0,0 +1,166 @@
+
+Based on his research of 100+ startups, Ellis believes 40% answering "very disappointed" is a strong signal of product-market fit.
+
+![How to calculate PMF score](../images/blog/pmf-surveys/pmf-score.png)
+
+## How to drive product-market fit with PMF surveys
+
+### Step 1: Survey informed users
+
+The first step is to learn more about your customers and what they think of your app. Do this by creating a [survey](/surveys) with the following questions:
+
+1. How would you feel if you could no longer use our product?
+   
+2. What type of people do you think would most benefit from it?
+
+3. What is the main benefit you receive from it?
+
+4. How can we improve our app for you?
+
+Each question plays an important and specific role – we'll explain how later. 
+
+In the meantime, to receive the best feedback, send the survey to users who have recently had meaningful interactions with your app. This means users that have:
+
+- experienced the core value-prop of your product.
+- used your product at least twice.
+- used your product in the last two weeks.
+
+You can begin analyzing the results once you have at least 30 responses. 
+
+### Step 2: Study your best customers
+
+Start by examining the users who would be "very disappointed" if they could no longer use your product. There are two things you need to learn about them:
+
+1. Who they are
+
+2. Why they love your product
+
+To find the answers, look at their responses to the questions `What is the main benefit you receive from our product?` and `What type of people do you think would most benefit from it?`.
+
+Then, use this information to create a [user persona](/product-engineers/how-to-create-user-personas) of your best supporters. Users who match this persona are the ones you need to improve your app for and you should focus your attention solely on them. In the next step we explain how to do this.
+
+![Personas of survey responders](../images/blog/pmf-surveys/personas.png)
+<Caption>Focus your attention only on users that match your user persona</Caption>
+
+### Step 3: Find your "on-the-fence" users
+
+Direct your attention to the users who responded with "somewhat disappointed". In particular, focus on the ones that:
+
+- match the user persona you created in the previous step, and
+- gave similar responses like your best supporters when asked about the main benefit they receive from your app.
+
+It's clear that these users are seeing the value in your app, but something is holding them back from becoming passionate users. Usually, only one or two things are in their way, so these users will be the easiest to convert into fanatics who champion your product.
+
+To find out what their problems are, look at their response to the question `"How can we improve our app for you?" and make a note of the common themes.
+
+### Step 4: Reprioritize your roadmap
+
+Now that you understand your app's most important benefits and missing features, the next step is to reprioritize your roadmap. 
```

Comment:
The last two steps didn't have to do with understanding "your app's most important benefits and missing features", they were about understanding users it seems. I feel like you're missing a step or part of a step here. 

Maybe adjusting the titles of step 3 and 4 would help.